lib/pf/ConfigStore/DNSFilters.pm

```diff
@@ -0,0 +1,54 @@
+package pf::ConfigStore::DNSFilters;
```

Comment:
I hate myself even more now that I know you hate it too.

I guess whats left to do is to go cry in my shower alone